xhr/resources/headers-some-are-empty.asis

```diff
@@ -0,0 +1,6 @@
+HTTP/1.0 200 MEH
+HEYA:	 
+HEYA: 1
+HEYA:
```

Comment:
Should we have a test that \C is not removed, since apparently some rules are going to have special handling of it (At least if we can't get rid of that)?

Reply:
Yeah good call, Safari fails that looks like.